xcode/Subconscious/Shared/Components/Detail/Detail.swift

```diff
@@ -1463,23 +1389,28 @@ struct DetailModel: ModelProtocol {
             )
             return Update(state: state)
         }
+
         let to = from.withAudience(audience)
+
         let fx: Fx<DetailAction> = environment.data.moveEntryAsync(
-            from: from.toEntryLink(title: state.headers.title),
-            to: to.toEntryLink(title: state.headers.title)
-        )
-        .map({ receipt in
+            from: from,
+            to: to
+        ).map({ receipt in
```

Comment:
I'm transitioning to this chaining style piecemeal, because unfortunately Xcode likes to bork the whitespace of the chaining style we used to use.